web/styles/settings.css

```diff
@@ -853,11 +853,16 @@ input[type="checkbox"] {
         &:disabled {
             cursor: not-allowed;
             background-color: hsl(0deg 0% 93%);
-            opacity: 0.7;
```

Comment:
I am not sure why you have changed this, but it would be better to have this in a separate commit with explaining the reason for the change.

Reply:
This was to fix color and make it look like other dropdowns. 

Note that now, I have removed this whole `:disabled` pseudo-selector block since the other two rules were repeated -- they are already defined by the `:disabled` selector on base `.dropdown-widget-button` class and removing them doesn't have any effect.

| Before | After |
|--------|--------|
| ![image](https://github.com/zulip/zulip/assets/133781250/397de4e5-fbae-4861-8cd9-b5fa178bc2c2) | ![image](https://github.com/zulip/zulip/assets/133781250/a853a7e8-fed3-474d-8c94-2199c50fe89e) | 

Actually I think we might also need a #design discussion to decide which of these styles to actually keep for the dropdowns?

Reply:
The color on the right is fine. We can adjust later on if we decide we don't like it, but I think it's clear enough that the field is disabled, and it's easier to read.

Reply:
Update on this, the darker text color was actually due to a Firefox issue. [See below](https://github.com/zulip/zulip/pull/30490#issuecomment-2204522467). These were actually consistent on Chrome.

I've for now updated these fields to look like in Firefox like they do in Chrome (bottom left in above image).